plugins/statistics.koplugin/main.lua

```diff
@@ -1142,11 +1152,20 @@ function ReaderStatistics:getBookStat(id_book)
     end
     local conn = SQ3.open(db_location)
     local sql_stmt = [[
-        SELECT title, authors, notes, highlights, pages, last_open
+        SELECT title, authors, pages, last_open, highlights, notes
         FROM book
         WHERE id = '%s'
     ]]
-    local title, authors, notes, highlights, pages, last_open = conn:rowexec(string.format(sql_stmt, id_book))
+    local title, authors, pages, last_open, highlights, notes = conn:rowexec(string.format(sql_stmt, id_book))
+
+    -- Due to some bug, some books opened around April 2020 might
```

Comment:
Although you'll see it in git blame, might be easier to include a reference to this PR/the issue in the comment?